locations/spiders/ultra_liquors.py

```diff
@@ -0,0 +1,30 @@
+import scrapy
+
+from locations.categories import Extras, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class UltraLiquorsSpider(scrapy.Spider):
```

Comment:
All results returned are in South Africa, hence this spider should be renamed to have the `ZA`/`_za` suffix.